src/google_apps_clj/google_calendar.clj

```diff
@@ -11,9 +11,10 @@
            (com.google.api.services.calendar Calendar
                                              Calendar$Builder
                                              Calendar$Events$Insert
-                                             CalendarScopes)))
+                                             CalendarScopes)
+           (com.google.api.client.googleapis.auth.oauth2 GoogleCredential)))
 
-(t/ann build-calendar-service [cred/GoogleCtx -> Calendar])
+(t/ann build-calendar-service [(t/U cred/GoogleCtx GoogleCredential) -> Calendar])
```

Comment:
That's a good idea.  `Credential` is already a Thing™ in the Google library code so I may go with something like `GoogleAuth` or just `Auth` instead to avoid confusion.